modules/reference/appvms/chromium.nix

```diff
@@ -71,6 +71,11 @@ in
       microvm.devices = [ ];
 
       ghaf.reference.programs.chromium.enable = true;
+      ghaf.storagevm = {
+        enable = true;
+        name = "business";
+        directories = [ "/home/${config.ghaf.users.accounts.user}/.config" ];
```

Comment:
seems to get owned by `root` and not writable. If this PR requires full clean install (instead of `nixos-rebuild ... switch` from zfs-enabled ghaf install, we must mention it in the PR description Testing-section.
```
[ghaf@chromium-vm:~/.config]$ mkdir puuppa
mkdir: cannot create directory ‘puuppa’: Permission denied

[ghaf@chromium-vm:~/.config]$ cd ..

[ghaf@chromium-vm:~]$ ls -lah
total 512
drwxr-xr-x 3 root root 60 Aug 20 13:33 .
drwxr-xr-x 3 root root 60 Aug 20 13:33 ..
drwxr-xr-x 2 root root  2 Aug 20 11:24 .config
```